webapp/src/components/properties/user/user.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react'
+import Select from 'react-select'
+
+import {CSSObject} from '@emotion/serialize'
+
+import {IUser} from '../../../user'
+import octoClient from '../../../octoClient'
+
+import './user.scss'
+
+type Props = {
+    value: string,
+    onChange: (value: string) => void,
+}
+
+const UserProperty = (props: Props): JSX.Element => {
+    const [users, setState] = useState<Array<IUser>>([])
+
+    useEffect(() => {
+        if (users.length === 0) {
+            const getWorkspaceUsers = async () => {
+                const workspaceUsers = await octoClient.getWorkspaceUsers()
+                setState(workspaceUsers)
+            }
+
+            getWorkspaceUsers()
+        }
+    })
+
+    let value: IUser | undefined
+
+    if (props.value) {
+        value = users.find((user) => user.id === props.value)
+    }
+
+    return (
+        <Select
+            options={users}
+            isSearchable={true}
+            isClearable={true}
+            backspaceRemovesValue={true}
+            className={'user-picker'}
+
+            styles={{
```

Comment:
Should these be coming from a constant, rather than setting them here. Could they be reused?

Reply:
Yes, they would serve well as base styles for React Select. I've updated it now.